lib/getAuthenticatorFromEnvCloudantExtension.ts

```diff
@@ -14,10 +14,15 @@
  * limitations under the License.
  */
 
-import { Authenticator, getAuthenticatorFromEnvironment, readExternalSources } from "ibm-cloud-sdk-core";
-import { CouchdbSessionAuthenticator } from "../auth";
+import {
+  Authenticator,
+  getAuthenticatorFromEnvironment,
+  readExternalSources,
+} from 'ibm-cloud-sdk-core';
+// eslint-disable-next-line node/no-missing-import
```

Comment:
Could not we turn off this? Do we use it in other places? Now it seems to me we have to turn this off manually in many places.

Reply:
I removed all possible global ignores. Ie. `overrides` only has entries for generated files and every other disable is local and is only there when necessary. If you think it makes code too noisy, I can move them to the config files. Wdyt?

Reply:
> I can move them to the config files. Wdyt?

If that does not come with more files to modify than now, why not?

Reply:
> why not

The locality aspect. Instead of having some ignores "far away" from the "violation", it's right there and obvious to everyone what's going on.

Reply:
> This rule checks the file paths of import and export declarations. If the file paths don't exist, this reports these.

Do the paths of the import exist? I guess so. Based on [this issue comment](https://github.com/microsoft/vscode-eslint/issues/1097#issuecomment-717272327) setting the `eslint.runtime` could help. Please let me know if I miss something here.

Reply:
I think I got this: 85ea6ee . What do you think?

Reply:
Much better, thank you!